Locker/Locker/Locker.swift

```diff
@@ -35,6 +35,7 @@ public class Locker: NSObject {
     #endif
     }
 
+    // swiftlint:disable:next identifier_name
     public static var deviceSupportsAuthenticationWithBiometrics: BiometricsType {
```

Comment:
```
    public static var supportedBiometricAuthentication: BiometricsType {
```
And remove swiftlint rule disabling. Once this is done, don't forget to update readme!